bliss/reporting.py

```diff
@@ -184,7 +184,7 @@ def match_by_locs(true_locs, est_locs, slack=1.0):
     locs1 = true_locs.view(-1, 2)
     locs2 = est_locs.view(-1, 2)
 
-    # entry (i,j) is l1 distance between of ith loc in locs1 and the jth loc in locs2
+    # entry (i,j) is l-infinity distance between of ith loc in locs1 and the jth loc in locs2
```

Comment:
This only applies to locs_err_l_infty. We should either expand this comment or delete it altogether to avoid confusion

Reply:
locs_abs_diff is still the L1 distance

Reply:
good point, I deleted the comment